src/popup/App.vue

```diff
@@ -19,7 +19,7 @@
         @click.self="showSidebar = false"
         data-cy="menu-overlay"
       >
-        <SidebarMenu @closeMenu="showSidebar = false" />
+        <SidebarMenu @close="showSidebar = false" />
```

Comment:
Should we change this naming in the scope of this PR?

Reply:
The original eslint error was:
> /superhero-wallet/src/popup/router/components/SidebarMenu.vue
  118:18  error  Custom event name 'closeMenu' must be kebab-case  vue/custom-event-name-casing

So, I have to update it and at the same time, I thought that the event emitted inside the component shouldn't contain the component name. Then I decided to don't edit this twice 🙃